src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -1021,7 +1021,7 @@ public String DependsOnNETStandard
         /// been outputted in MSB3277. Otherwise empty.
         /// </summary>
         [Output]
-        public ITaskItem[] UnresolvedAssemblyConflicts => _unresolvedConflicts.ToArray();
+        public ITaskItem[] UnresolvedAssemblyConflicts => ReferenceTable.NormalizeAsArray(_unresolvedConflicts);
```

Comment:
Is this only accessed once? because if it is read twice or more, we'll do the work every time.

Reply:
It could be accessed many times, depending on the user's use case. I can cache it.